lib/ex_unit/lib/ex_unit/runner.ex

```diff
@@ -341,12 +341,12 @@ defmodule ExUnit.Runner do
   end
 
   defp failed(:error, %ExUnit.MultiError{errors: errors}, _stack) do
-    {
-      :failed,
+    errors =
```

Comment:
just wondering if this could this be `formatted_errors` perhaps?

would reusing the `errors` variable name cause any issues?

Reply:
(still pretty new to Elixir)

Reply:
The errors are not "formatter", rather "normalized" I would say. Using `errors` doesn't cause any issues and rebinding variables (especially in these simple cases) is a very common practice in Elixir code :).

Reply:
cheers!